guides/common/modules/con_using-ipxe-to-reduce-provisioning-times.adoc

```diff
@@ -0,0 +1,12 @@
+[id="using-ipxe-to-reduce-provisioning-times_{context}"]
+= Using iPXE to reduce provisioning times
+
+iPXE is an open-source network-boot firmware.
+It provides a full PXE implementation enhanced with additional features, such as booting from an HTTP server.
+For more information, see https://ipxe.org[ipxe.org].
```

Comment:
I'd like to see a more descriptive link label. Also, I think we mentioned recently in another PR that using the link macro is better, right?

```suggestion
For more information about iPXE, see the link:https://ipxe.org[iPXE website].
```

See also "Do not use bare URLs for links." in [SSG](https://redhat-documentation.github.io/supplementary-style-guide/#external-links). That guide also uses link macros.